packages/db/index.d.ts

```diff
@@ -38,7 +38,7 @@ export class DBClient {
   getMetricsValue (key: string): Promise<{ total: number }>
   upsertUser (user: UpsertUserInput): Promise<UpsertUserOutput>
   getUser (issuer: string): Promise<UserOutput>
-  getUsedStorage (userId: number): Promise<number>
+  getUsedStorage (userId: number): Promise<Record<string, string>>
```

Comment:
Given we know the exact keys of the object returned here, I wonder if we should be more explicit with the return type

Reply:
Good point. Have made the return type explicit.

Reply:
Thanks!